### DISCUSSION TITLE
Integration of Remix App with New-Relic

### DISCUSSION THREAD

**Discussion by ram2104** [remix-run/remix#5233] [Q&A]
Hi,

I tried integrating the new relic into the remix app but couldn't find better integration docs. I did try to incorporate it by looking [nextjs](https://newrelic.com/blog/how-to-relic/nextjs-monitor-application-data) integration docs, but It didn't help. 

I also tried the node-based integration, even in the browser, because we are using the node-specific module on the client side, which is expected.

Any suggestion 

Cheers :)

**Comment by lukeaskew** [comment]
I took a stab at this with no luck. I am running serverless and couldn't get the lambda layer and Remix app code to work together.

I'd love to continue working on this if anyone with relevant expertise is available to collaborate.
